submissions/AndriyM/friends-app/style.css

```diff
@@ -0,0 +1,344 @@
+	color: #212529;
+	background-color: #f2f4f5;
+	background-clip: padding-box;
+	border: 1px solid #fff;
+	border-radius: 0.25rem;
+	transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+}
+
+.form__input::placeholder {
+	color: #444749;
+	opacity: 0.4;
+}
+
+.form__input:focus {
+	color: #212529;
+	background-color: #fff;
+	border-color: #bdbdbd;
+	outline: 0;
+	box-shadow: 0 0 0 0.1rem rgb(118 90 225);
+}
+
+.form__sort-age,
+.form__sort-name {
+	display: flex;
+	margin-bottom: 25px;
+	gap: 20px;
+	align-items: center;
+	font-size: 18px;
+	font-weight: 600;
+	color: #221f1f;
+}
+
+.svg-age,
+.svg-name {
+	width: 36px;
+	height: 36px;
+	cursor: pointer;
+}
+
+.svg-age:nth-last-child(),
+.svg-name:nth-last-child() {
+	margin-right: 0px;
+}
+
+.btn {
+	margin-left: auto;
+	margin-right: auto;
+	padding: 8px 16px;
+	border-radius: 5px;
+	background-color: rgb(118 90 225);
+	transition: all 0.25s cubic-bezier(0.4, 0, 0.2, 1);
+	text-align: center;
+	color: #fff;
+	text-transform: uppercase;
+	border: 0;
+	text-decoration: none;
+	cursor: pointer;
+	font-size: 18px;
+	line-height: 24px;
+	font-weight: 400;
+	min-width: 150px;
+	box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2),
+		0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 5px 0 rgba(0, 0, 0, 0.12);
+}
+.form__clear {
+	display: block;
+}
+
+.form__clear:hover,
+.form__clear:focus {
+	background-color: rgb(145 122 230);
+}
+
+.form__label {
+	font-size: 18px;
+	font-weight: 600;
+	color: rgb(118 90 225);
+	text-shadow: 3px 1px 13px rgb(118 90 225);
+}
+
+.form__gender,
+.form__age {
+	margin-bottom: 25px;
+}
+.form__gender-label {
+	font-weight: 400;
+	font-size: 18px;
+	color: #221f1f;
+	cursor: pointer;
+	margin-right: 15px;
+}
+.form__gender-label:nth-last-child() {
+	margin-right: 0px;
+}
+.form__gender-radio {
+	cursor: pointer;
+}
+.form__gender-text,
+.form__age-label {
+	margin-bottom: 8px;
+	font-weight: 600;
+	font-size: 18px;
+	color: #221f1f;
+}
+
+.form-filter {
+	position: relative;
+	z-index: 88;
+	justify-content: center;
+	background-color: rgb(118 90 225);
+	cursor: pointer;
+	writing-mode: vertical-lr;
+	transform: rotate(180deg);
+	padding: 10px 6px;
+	border-radius: 10px;
+	font-size: 23px;
+	color: #fff;
+	text-transform: uppercase;
+	letter-spacing: 18px;
+	font-weight: 700;
+}
+
+.overlay {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	background: rgba(0, 0, 0, 0.7);
+	z-index: 50;
+	opacity: 1;
+	transition: opacity 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.overlay.is-hidden {
+	opacity: 0;
+	pointer-events: none;
+}
+
+/* ---------------- adaptive ----------------- */
+
+@media (max-width: 1160.98px) {
+	.card {
+		flex-direction: column;
+	}
+	.card__age {
+		top: 11px;
+		right: 11px;
+	}
+}
+
+@media (max-width: 799.98px) {
+	.form-filter {
+		display: block;
+	}
+}
+
+@media (min-width: 800.98px) {
+	.form-filter {
+		display: none;
+	}
+	.main__form {
+		display: block;
+		width: 300px;
+		height: 100%;
+	}
+}

```

Comment:
Responcive css - it is cool!!!
But why your pixel size fractional? Please, use something like `800` or `801`. Thanks)